submissions/Vlada-khom/task-js-oop/app.js

```diff
@@ -0,0 +1,128 @@
+const X_START_PLAYER = 200;
+const Y_START_PLAYER = 400;
+const Y_LINE_VICTORY = 0;
+const CELL = {
+  width: 100,
+  height: 85,
+};
+const VISIBLE_BORDER = {
+  width: 500,
+  height: 400,
+};
+
+const ENEMY_SPEED = {
+  minSpeed: 60,
+  maxSpeed: 200,
+};
+
+const ROW_FOR_ENEMIES = {
+  one: 230,
+  two: 145,
+  three: 60
+}
+
+let count = 0;
+let counter = document.createElement("div");
+document.body.append(counter);
+
+const Player = function (x, y, count) {
+  this.x = x;
+  this.y = y;
+  this.sprite = "images/char-boy.png";
+  this.count = count;
+};
+
+Player.prototype.update = function () {
+  if (this.y < Y_LINE_VICTORY) {
+    alert("You WINNER!");
+    this.x = X_START_PLAYER;
+    this.y = Y_START_PLAYER;
+    allEnemies.forEach((el) => (el.xMove = el.x));
```

Comment:
[This comment](https://github.com/kottans/frontend-2022-homeworks/pull/255#discussion_r955001848) remains unaddressed.